cinema/views.py

```diff
@@ -0,0 +1,49 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    match request.method:
+        case "GET":
+            movies = Movie.objects.all()
+            serializer = MovieSerializer(movies, many=True)
+            return Response(serializer.data, status=status.HTTP_200_OK)
+
+        case "POST":
+            serializer = MovieSerializer(data=request.data)
+
+            if serializer.is_valid(raise_exception=True):
```

Comment:
if you use serializer.is_valid(raise_exception=True) you shouldn't use Response with 400 status, it does it automatically

Reply:
Fixed

Reply:
`if` is useless here as well

Reply:
Fixed